dxtbx/format/FormatMultiImage.py

```diff
@@ -177,7 +176,8 @@ def get_imageset(Class,
       format_kwargs = {}
 
     # If we have no specific format class, we need indices for number of images
-    if Class == FormatMultiImage:
+    from dxtbx.format.FormatMultiImageJIT import FormatMultiImageJIT
+    if Class in [FormatMultiImage, FormatMultiImageJIT]:
```

Comment:
I think this case should be identified by a property of the class, not by yet another import and comparing to a list.